io.github.face_hh.Napture.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "io.github.face_hh.Napture",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "napture",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--own-name=org.bussin.napture"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/napture/napture/cargo"
+        }
+    },
+    "modules": [
+        {
+            "name": "lua",
```

Comment:
Why not using the shared-modules https://github.com/flathub/shared-modules/tree/master/lua5.4